BIDScramble/bidscramble/scramble_fif.py

```diff
@@ -35,10 +20,9 @@ def scramble_fif(inputdir: str, outputdir: str, select: str, bidsvalidate: bool,
     for inputfile in tqdm(inputfiles, unit='file', colour='green', leave=False):
 
         # Figure out which reader function to use, fif-files with time-series data come in 3 flavours
-        fiffstuff = mne.io.show_fiff(inputfile)
-        isevoked  = re.search('FIFFB_EVOKED', fiffstuff) is not None
-        isepoched = re.search('FIFFB_MNE_EPOCHS', fiffstuff) is not None
-        israw     = not isepoched and not isevoked
+        isevoked  = not all(len(l) == 0 for l in mne.io.show_fiff(inputfile,output=list,tag=104))
```

Comment:
Can't you just do:
```
sraw = any(mne.io.show_fiff(inputfile, output=list, tag=102))
etc
```

Reply:
yes, good point. My new best friend suggested this, and I did not think it through. I changed it.